### PR TITLE
Realtime Tuning

### PR DESCRIPTION
**Pull request by kunalsheth**

### CODE REVIEW

File: eighteen/src/main/kotlin/com/lynbrookrobotics/kapuchin/subsystems/drivetrain/DrivetrainComponent.kt [hunk 1]
```diff
@@ -46,4 +48,8 @@ class DrivetrainComponent(hardware: DrivetrainHardware) : Component<DrivetrainCo
         leftLazyOutput(value.left)
         rightLazyOutput(value.right)
     }
+
+    init {
+        if (clock is Ticker) clock.realtimeChecker(hardware.jitterPulsePin::set, { hardware.jitterReadPin.period.Second })
```
Comment (kunalsheth) [inline eighteen/src/main/kotlin/com/lynbrookrobotics/kapuchin/subsystems/drivetrain/DrivetrainComponent.kt]: `jitterReadPin` is a WPI [`Counter`](http://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/Counter.html).
`Counter` has as `getPeriod()` method which returns a `double` in seconds.
In Kotlin, we call this method with `jitterReadPin.period`, so we have to convert the result to a `Quan` ourselves.

Comment (akainth015) [reply eighteen/src/main/kotlin/com/lynbrookrobotics/kapuchin/subsystems/drivetrain/DrivetrainComponent.kt]: Yep, realized this after I got the IDE to work properly.

File: architecture/src/main/kotlin/com/lynbrookrobotics/kapuchin/timing/Thread.kt [hunk 2]
```diff
@@ -3,6 +3,15 @@ package com.lynbrookrobotics.kapuchin.timing
 import com.lynbrookrobotics.kapuchin.logging.Named
 import kotlinx.coroutines.CoroutineScope
 
+/**
+ * Represents the importance of something being executed
+ *
+ * Intended for specifying JVM and OS thread priorities. `RealTime` is the highest priority possible.
+ */
+enum class Priority {
```
[the remark this answers is not in the document]
Comment (akainth015) [reply architecture/src/main/kotlin/com/lynbrookrobotics/kapuchin/timing/Thread.kt]: Yep, that's good, I need to walk through our structure

File: simulated/build.gradle [hunk 3]
```diff
@@ -20,7 +20,7 @@ compileKotlin { kotlinOptions.jvmTarget = "1.8" }
 compileTestKotlin { kotlinOptions.jvmTarget = "1.8" }
 
 test {
-//    jvmArgs '-verbose:class'
+    jvmArgs '-verbosegc', '-Xms100m', '-Xmx100m'
```
[the remark this answers is not in the document]
Comment (akainth015) [reply simulated/build.gradle]: Ok,  and what about the verbose collection? Do all testers want to have detailed logging? Maybe a comment is called for.

**Pull request merged**